Source/WebCore/style/values/color/StyleColorLayers.cpp

```diff
@@ -92,25 +92,18 @@ bool containsCurrentColor(const ColorLayers& colorLayers)
     });
 }
 
-// MARK: - Serialization
-
-void serializationForCSS(StringBuilder& builder, const CSS::SerializationContext& context, const ColorLayers& colorLayers)
-{
-    CSS::serializationForCSSColorLayers(builder, context, colorLayers);
-}
-
-String serializationForCSS(const CSS::SerializationContext& context, const ColorLayers& colorLayers)
-{
-    StringBuilder builder;
-    serializationForCSS(builder, context, colorLayers);
-    return builder.toString();
-}
-
 // MARK: - TextStream
 
 WTF::TextStream& operator<<(WTF::TextStream& ts, const ColorLayers& colorLayers)
 {
-    return ts << serializationForCSS(CSS::defaultSerializationContext(), colorLayers);
+    ts << "color-layers(";
+
+    if (colorLayers.blendMode != BlendMode::Normal)
+        ts << colorLayers.blendMode << ", "_s;
+    ts << colorLayers.colors;
+    ts << ')';
```

Comment:
```suggestion
    ts << colorLayers.colors << ')';
```
Not sure this is as important as in `StringBuilder::append`, where doing multiple in a single call is more efficient, but it does at least seem idiomatic to combine into a single line when possible.